cms/djangoapps/contentstore/views/course.py

```diff
@@ -1156,6 +1156,13 @@ def settings_handler(request, course_key_string):  # lint-amnesty, pylint: disab
             verified_mode = CourseMode.verified_mode_for_course(course_key, include_expired=True)
             upgrade_deadline = (verified_mode and verified_mode.expiration_datetime and
                                 verified_mode.expiration_datetime.isoformat())
+
+            date_placeholder_format = configuration_helpers.get_value_for_org(
+                course_module.location.org,
+                'SCHEDULE_DETAIL_FORMAT',
+                settings.SCHEDULE_DETAIL_FORMAT
+            ).upper()
```

Comment:
I asked around what should we do about using site configurations since it's set to be deprecated, the initial response is to not add more technical debt given this will need to be removed once site config it's no longer used.

Can we use django settings here?